src/Snippet.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+
```

Comment:
I'm assuming the two blank lines here are because of some formatting rule?

Reply:
Nah no good reason for that, I think it was related to me trying to get tslint to not complain